File: BubbleSort.java
public class BubbleSort {
    
        public static void main(String[] args) {
               
                int intArray[] = {5,90,35,45,150,3};                 
                
                bubbleSort(intArray);                            
                
                for(int i=0; i < intArray.length; i++){
                        System.out.print(intArray[i] + " ");
                }
                System.out.println();

        } 
        public static void bubbleSort(int[] intArray) {               
                int n = intArray.length;
                int temp = 0;               
                for(int i=0; i < n; i++){
                        for(int j=1; j < (n-i); j++){                               
                                if(intArray[j-1] > intArray[j]){
                                        //swap the elements!
                                        temp = intArray[j-1];
                                        intArray[j-1] = intArray[j];
                                        intArray[j] = temp;
                                }
                               
                        }
                }


        }
}

File: bubblesort.c
#include <stdio.h>
#define MAX 10

void bubbleSort(int list[]) {
   int temp;
   int i,j;
	
   for(i = 0; i < MAX-1; i++) { 
      for(j = 0; j < MAX-1-i; j++) {  			
         if(list[j] > list[j+1]) {
            temp = list[j];
            list[j] = list[j+1];
            list[j+1] = temp;                   
         }			
      }
   }	
}

main() {
   int list[MAX] = {1,81,4,6,-1,34,5,2,7,9};
   bubbleSort(list);
   int i =0;
   for(i = 0; i < MAX; i++){
      printf("%d ",list[i]);
   }
   printf("\n");      
}
